### PR TITLE
Automatically label pull requests

### PR DESCRIPTION
**Pull request by jchristgit**

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+"group: docs":
+  - changed-files:
+      - any-glob-to-any-file: 'docs/*'
+
+"group: github actions":
+  - changed-files:
+      - any-glob-to-any-file: '.github/workflows/*'
+
+"group: kubernetes":
+  - changed-files:
```
Comment (shtlrs) [inline .github/labeler.yml:18]: Do these labels need to exist before it can create them? Or. Will it add them auto?

Comment (jb3) [reply .github/labeler.yml:18]: It will create them

**Review by jb3 — APPROVED** [review]
Looks great!

**Comment by jb3** [comment]
Merged manually in 134a90f. Thanks!

**Pull request closed without merging**